### DISCUSSION TITLE
SMTP 500 Error

### DISCUSSION THREAD

**Discussion by ghost** [modoboa/modoboa#2205] [Help] [answered]
I recently installed a Modoboa server, and everything's perfect... minus (what seems to be) an SMTP error.

This has led to my mail getting rejected by some big-name email providers. 

Any ideas?

```
Reporting-MTA: dns; mail.eml.pm
X-Postfix-Queue-ID: 50A17416A4
X-Postfix-Sender: rfc822; jacob@eml.pm
Arrival-Date: Thu, 25 Mar 2021 19:08:02 +0100 (CET)

Final-Recipient: rfc822; example@bt.com
Original-Recipient: rfc822;example@bt.com
Action: failed
Status: 5.5.0
Remote-MTA: dns; smtpe1.intersmtp.com
Diagnostic-Code: smtp; 550 5.5.0 550 5.7.1 Requested action not taken: message
    refused
```

This is the domain's DNS things in Modoboa (all green):

![image](https://user-images.githubusercontent.com/75087999/112524964-4e1cfc80-8d98-11eb-8acc-f5d449dfa7ef.png)

**Comment by tonioo** [comment]
Difficult to analyze without more information... "message refused" is not really clear! Do you if this mail provider has a FAQ where you can check the meaning of error codes?

**Reply by ghost** [reply]
I typed my mail host (mail.eml.pm) in on one of those email server checker websites and I saw that it came back with this same 500 SMTP error... that was a few days ago and I can't find it in my history, but I'm looking for that site/similar now to see if I get the same error - and see if there's any more details.

**Reply by ghost** [reply]
Upon further investigation, it looks to be a firewall error or something of that sort. I've tried both with and without a firewall and the error still happens. This link (https://www.knownhost.com/wiki/email/troubleshooting/error-numbers/500) says it may be an error with a firewall/antivirus, which sounds to me like Hetzner has an issue, but I've heard of many people running mail servers with Hetzner without issue.

Is it likely to be a Hetzner/VPS provider issue?

**Comment by ghost** [comment, marked as answer]
I've found the issue - it was two fold.

First the tutorial I followed told me that Ubuntu 20.04 was supported, which it isn't - I've now reinstalled using Debian 10.

It turns out `bt.com` has very stringent and odd mail delivery policies, which is why my mail was rejected.
